plugins/uefi-capsule/fu-uefi-bootmgr.c

```diff
@@ -327,6 +327,7 @@ fu_uefi_bootmgr_bootnext(FuDevice *device,
 			}
 			use_fwup_path = FALSE;
 		} else if ((flags & FU_UEFI_BOOTMGR_FLAG_USE_SHIM_FOR_SB) > 0) {
+			g_debug("couldn't find shim binary at %s\n", shim_app);
```

Comment:
can this go into an inhibit perhaps? then we can see the message as a user.

Reply:
I thought about that, but then we need an inotify on the directory to be able to restore it.  Since the partition is mounted and unmounted dynamically in some circumstances that is not a good idea

Reply:
Could potentially just add it into the error message I suppose.

Reply:
> Could potentially just add it into the error message I suppose

Yup, this I think .

Reply:
fixed